### PR TITLE
Landing page UI/main

### PR DESCRIPTION
**Pull request by adundarov**

### CODE REVIEW

**Comment by jspaniac** [comment]
There are some errors in the console regarding
"MUI: You are providing an onClick event listener to a child of a button element."
We could push it for the demo and open an issue or we could try and fix it now. I'll go ahead and start investigating later tonight

**Review by jspaniac — APPROVED** [review]
LGTM! I went ahead and fixed the console errors I was seeing and also added in a quick fix for some issues in the results page (back button & wrap around)

File: frontend/my-app/src/Pages/LandingPage.tsx [hunk 1]
```diff
@@ -69,26 +78,144 @@ const LandingPage: React.FC = () => {
...
+            alignSelf: 'start',
+            width: 'auto',
+            minWidth: 4 / 5,
+            mb: 'auto',
+            mt: 5,
+            mx: 'auto'
+          }}
+          >
+            <Toolbar>
+              <Button color='secondary' variant='contained' onClick={handleLogoutClick}>Logout</Button>
```
Comment (zshafe) [inline frontend/my-app/src/Pages/LandingPage.tsx:100]: NIT: I would prefer if some of the style here was cleaned up. For example:
```
<Button color='secondary' variant='contained' onClick={handleLogoutClick}>Logout</Button>
```
should look like
```
<Button
     color='secondary'
     variant='contained'
     onClick={handleLogoutClick}>
          Logout
</Button>
```

File: frontend/my-app/src/Pages/LandingPage.tsx [hunk 2]
```diff
@@ -69,26 +78,144 @@ const LandingPage: React.FC = () => {
...
+              >
+                {getCheckedFriends().map((user, index) => (
+                  <IconButton color="primary" aria-label="upload picture" component="label"
+                  sx={{
+                    width: 50,
+                    height: 50,
+                    backgroundColor: 'primary.dark',
+                    '&:hover': {
+                      backgroundColor: 'success.main',
+                      opacity: [0.8, 0.8, 0.8]
```
Comment (zshafe) [inline frontend/my-app/src/Pages/LandingPage.tsx]: Is there a way to condense these styling aspects to something smaller and resuable?

**Review by zshafe — CHANGES_REQUESTED** [review]
just some style that would be nice to clean up but otherwise LGTM

**Review by zshafe — APPROVED** [review]

**Pull request merged**